src/STM32RTC.cpp

```diff
@@ -154,10 +162,18 @@ void STM32RTC::getPrediv(int8_t *predivA, int16_t *predivS)
   * @param  predivS: Synchronous prescaler value. Reset value: -1
   * @retval None
   */
+#if defined(STM32F1xx)
+void STM32RTC::setPrediv(int32_t predivA, int16_t predivS)
+{
+  UNUSED(predivS);
+  RTC_setPrediv(predivA, 0);
```

Comment:
```suggestion
void STM32RTC::setPrediv(uint32_t predivA, int16_t dummy)
{
  UNUSED(dummy);
  RTC_setPrediv(predivA, 0);
```

Reply:
yes, predivS is a dummy param for stm32F1 but then changing the param name does not match the description of the function anymore